README.md

```diff
@@ -223,5 +223,6 @@ The following Environment Variables must be set in your shell prior to running a
 - `AZDO_DOCKERREGISTRY_SERVICE_CONNECTION_USERNAME`
 - `AZDO_GITHUB_SERVICE_CONNECTION_PAT`
 - `AZDO_TEST_AAD_USER_EMAIL`
+- `AZDO_TEST_AAD_GROUP_ID`
```

Comment:
`AZDO_TEST_AAD_GROUP_ID`  is for `azuredevops_group_entitlment` test, no need to document in this file

Reply:
This is to follow the pattern of ``AZDO_TEST_AAD_USER_EMAIL` which is used by `azuredevops_user_entitlement` test and a few others. If not documented here, anywhere else you'd recommend as a better place?

Reply:
No need to document this as this is used for AccTest.  The pre-check in test will throw an error if not set

Reply:
Well, the section it's documented to is called `## Environment variables for acceptance tests`, plus the `AZDO_TEST_AAD_USER_EMAIL` is also for AccTest only. I agree however README.md might not be a good place for such doc, but I'd suggest we keep `AZDO_TEST_AAD_GROUP_ID` here for consistency and move this whole section to a better place, for example `docs/contributing.md` sounds good for such docs.